converter/src/main/java/gov/cms/qpp/conversion/model/Node.java

```diff
@@ -319,17 +322,32 @@ public int getColumn() {
 	 *
 	 * @return The path.
 	 */
-	public String getPath() {
+	public String getOrComputePath() {
+		if (path == null) {
+			if (elementForLocation != null) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)